sound/soc/sof/topology.c

```diff
@@ -36,6 +36,8 @@
 #define VOLUME_FWL	16
 /* 0.5 dB step value in topology TLV */
 #define VOL_HALF_DB_STEP	50
+/* full volume for default values */
+#define VOL_ZERO_DB	(1 << 16)
```

Comment:
BIT(VOLUME_FWL) ?

Reply:
ok, I didn't even check/understand the previously defined values. Can change this.

Reply:
Could all add comment to VOLUME_FWL that define to 16 sets the volume linear gain value to use Qx.16 format.

It's out of scope of this PR but those macros VOL_TWENTIETH_ROOT_OF_TEN and VOL_FORTIETH_ROOT_OF_TEN could be re-calculated for a higher number of fractional bits (31) and shifted right by (31-VOLUME_FWL) to match the volume multiplier Q format. Then the Q format should be controllable in single macro.

Reply:
@singalsu added your comment to the VOLUME_FWL define.